runtime/oti/jextractnatives_api.h

```diff
@@ -43,18 +43,6 @@ extern "C" {
 #endif
 
 /* ---------------- jextractnatives.c ---------------- */
-
-/**
-* @brief
-* @param *env
-* @param obj
-* @param dumpObj
-* @param commandObject
-* @return void
-*/
-void JNICALL
-Java_com_ibm_jvm_j9_dump_extract_Main_doCommand(JNIEnv *env, jobject obj, jobject dumpObj, jstring commandObject);
```

Comment:
Please also remove the definition in `jextractnatives.c`, and the exports in `runtime/jextractnatives/CMakeLists.txt` and `runtime/jextractnatives/module.xml`.

That eliminates the only use of `void run_command(const char *)` (in `runtime/jextractnatives/jextractglue.c`). Since that source file contains no other definitions, it and `jextractnatives_internal.h` can be removed.